setup.cfg

```diff
@@ -26,6 +26,7 @@ install_requires =
     click==7.1.2
     click-option-group==0.5.2
     elasticsearch==7.11.0
+    genson==1.2.2 
```

Comment:
Why preferring `genson` over [`datamodel-code-generator`](https://github.com/koxudaxi/datamodel-code-generator/) as the latest supports JSON input data?

Reply:
`genson` is able to update it's JSON schema while receiving additional events while `datamodel-code-generator` only takes one `dict` or JSON schema and generates the corresponding `pydantic` model. They work in pair. We feed `genson` 100 events of source `browser` / type `seq_next` and `genson` generates a JSON schema that would match all these events. Then we use `datamodel-code-generator` to generate the corresponding `pydantic` model.

Reply:
Apropos, `genson` can be extended to include additional information in the JSON schemas (minimum/maximum values for integers, min-max length of strings, discover categorical values in strings using regex etc... But I'm hesitating to use these features at this stage as this could include some user data in the JSON schemas which should be avoided.

Reply:
Okay, the combination of the two tools makes sense, thanks for the clarification.

Reply:
So maybe we should add `datamodel-code-generator` to dependencies, so that the Docker image bundles tools required to generate models from JSON-serialized data?

Reply:
Good Idea) Thanks)